tools/question_converters/randexam/randexam2pl2.py

```diff
@@ -85,9 +85,9 @@ def log_array(arr, arr_name, dim_names):
     log(
         "{} array: ({})".format(
             arr_name,
-            ", ".join([
-                f"{dim_names[i]} = {arr.shape[i]}" for i in range(len(arr.shape))
-            ]),
+            ", ".join(
+                [f"{dim_names[i]} = {arr.shape[i]}" for i in range(len(arr.shape))]
```

Comment:
```suggestion
                f"{dim_names[i]} = {arr.shape[i]}" for i in range(len(arr.shape))
```
I don't think this is needed?